recipes/handyreadgenotyper/meta.yaml

```diff
@@ -1,31 +1,27 @@
 {% set name = "HandyReadGenotyper" %}
-{% set version = "0.1.18" %}
+{% set version = "0.1.20" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/AntonS-bio/HandyReadGenotyper/archive/{{ version }}.tar.gz
-  sha256: 70bac2fa184f770066952629975f7721cd601850c1572a55bd3189608767a9ea
+  sha256: 7e20556165a279f28f5862f38ef681d71b2a6176c72398b99dbce7089820b14b
 
 build:
-  number: 0
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vvv --no-deps --no-build-isolation --no-cache-dir
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   run_exports:
-      - {{ pin_subpackage('handyreadgenotyper', max_pin="x.x") }}
-  entry_points:
-    - classify = classify:main
-    - train = train:main
-    - genotyper_utilities = genotyper_utilities:main
+      - {{ pin_subpackage('handyreadgenotyper', max_pin="x.x") }}  
+  number: 0
 
 requirements:
   host:
     - pip
-    - python >=3.8,<3.9
+    - python >=3.8,<3.9.0a0
   run:
-    - python >=3.8,<3.9
+    - python >=3.8,<3.9.0a0
```

Comment:
Beware, you need to allow higher Python versions soon, Pandas 2.0 is the last one that supports 3.8

3.8 will be EOL soon, in like 4 months as well.

Reply:
The python version is the result of python version specified for an external dependency pysam. I'll look to change this in the future.